doc/source/Manual.rst

```diff
@@ -1859,7 +1859,7 @@ If you have Git_ installed on your development system,
 you can use it together with pip
 to install the latest version of |PyInstaller| directly::
 
-    pip install -e git://github.com/pyinstaller/pyinstaller.git#egg=PyInstaller
+    pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip
```

Comment:
This should still use `-e` to get an editable install. At least we should point to this option by listing teh commend with and without `-e`.